docs/vectorizer-quick-start.md

```diff
@@ -90,8 +90,8 @@ Now we can create and run a vectorizer. A vectorizer is a pgai concept, it proce
     SELECT ai.create_vectorizer(
          'blog'::regclass,
          loading => ai.loading_column('contents'),
-         destination => 'blog_contents_embeddings',
-         embedding => ai.embedding_ollama('nomic-embed-text', 768),
+         destination => ai.destination('blog_contents_embeddings'),
```

Comment:
```suggestion
         destination => ai.destination_table('blog_contents_embeddings'),
```

Reply:
also nit: but I'd put that as the last element to simulate the pipeline (it's the last output element)